test/production/terser-class-static-blocks/terser-class-static-blocks.test.ts

```diff
@@ -10,4 +12,21 @@ describe('terser-class-static-blocks', () => {
     const $ = await next.render$('/')
     expect($('p').text()).toBe('hello world')
   })
+
+  if (!isNextDeploy) {
```

Comment:
On next deploy test does it not fail since has no test in it?

Reply:
The test above is enough to not fail for deploy